# Timestamp.
std_msgs/Header header

# GPS coordinates where the sample was interesting.
float64 latitude
float64 longitude

float32 data
